config/deployer/config.ts

```diff
@@ -758,33 +751,6 @@ export const setupGlobals = async (config: Config) => {
   const txMap = await config.provider.set_map_config(mapCalldata);
```

Comment:
why do we set map config again if the map config is already set in the world config ?